apps/psp/docker-compose.yaml

```diff
@@ -8,12 +8,14 @@ services:
     environment:
       POSTGRES_PASSWORD: postgrespassword
   graphql-engine:
-    image: fedormelexin/graphql-engine-arm64:v2.1.0-beta.2
+    image: hasura/graphql-engine:v2.0.0-alpha.11
```

Comment:
do you remember what error you've encountered when using  fedormelexin/graphql-engine-arm64:v2.1.0-beta.2 image? xD

Reply:
I don't remember how much I bench pressed yesterday xD